src/types/full_block.py

```diff
@@ -37,7 +37,7 @@ class FullBlock(Streamable):
     foliage_sub_block: FoliageSubBlock  # Reward chain foliage data
     foliage_block: Optional[FoliageBlock]  # Reward chain foliage data (tx block)
     transactions_info: Optional[TransactionsInfo]  # Reward chain foliage data (tx block additional)
-    transactions_generator: Optional[Program]  # Program that generates transactions
+    transactions_generator: Optional[SerializedProgram]  # Program that generates transactions
```

Comment:
Should we also replace the Program within spend_bundle.py (CoinSolution) ? 
The spend bundle will get serialized and sent to a different process for validation (as well propagated to other peers).

Reply:
I'm a bit ignorant of this codebase still. I'm aiming to primarily just change the part that needs to be run by all full nodes, which I believe is validating full blocks and the mempool. The idea being that a smaller patch would be less error prone and would get us most of the speed-up benefits.

Basically, I'm trying to stay away from touching wallet logic.

Reply:
Good thinking @mariano54 . No, this won't change the form of the spend bundle, or the shape of the block program or its outputs. The hope is that this will be a drop in replacement for only the CLVM machinery, not any of the code run on the clvm.